services/healthcheck.js

```diff
@@ -0,0 +1,61 @@
+const healthcheck = require('@hmcts/nodejs-healthcheck');
+const os = require('os');
+const ioRedis = require('ioredis');
+const config = require('config');
+
+const outputs = require('@hmcts/nodejs-healthcheck/healthcheck/outputs');
+
+const { OK } = require('http-status-codes');
+const logger = require('logger');
+
+
+const client = ioRedis.createClient(
+  config.redis.url,
+  { enableOfflineQueue: false }
+);
+
+client.on('error', error => {
+  logger.trace(`Health check failed on redis: ${error}`, 'health_check_error');
+});
+
+const healthOptions = message => {
+  return {
+    callback: (error, res) => { // eslint-disable-line id-blacklist
+      if (error) {
+        logger.trace(`health_check_error: ${message} and error: ${error}`);
+      }
+      return !error && res.status === OK ? outputs.up() : outputs.down(error);
+    },
+    timeout: config.health.timeout,
+    deadline: config.health.deadline
+  };
+};
+
+const setup = app => {
+  healthcheck.addTo(app, {
+    checks: {
+      redis: healthcheck.raw(() => client.ping().then(_ => healthcheck.status(_ === 'PONG'))
```

Comment:
move redis to readiness? cc @adusumillipraveen

Reply:
Thanks @timja and @adusumillipraveen as per ticket says to include this redis in default health and readiness
https://tools.hmcts.net/jira/browse/SSCS-7300
I've included into both endpoints